deploy/olm-catalog/managed-api-service/1.1.0/integreatly.org_rhmiconfigs_crd.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
why is this 1.1.0 stuff in here?

Reply:
@philbrookes  Added 'rebalancePods' field to the csv, as requested by Craig, to turn this functionality on and off. Wasn't able to catch you at the time, but spoke to Paul and Craig and agreed on 1.1.0 as an appropriate version